x/ibc/core/02-client/keeper/client.go

```diff
@@ -17,6 +17,14 @@ import (
 func (k Keeper) CreateClient(
 	ctx sdk.Context, clientID string, clientState exported.ClientState, consensusState exported.ConsensusState,
 ) error {
+	params := k.GetParams(ctx)
+	if !params.IsAllowedClient(clientState.ClientType()) {
```

Comment:
This is ok.. We need to add explicity documentation for chains that they **must not** register on the codec two light clients which return the same thing for `ClientType()`, otherwise this check can be bypassed